### PR TITLE
Conda recipe for PheWAS R package - phenome wide association study

### PR DESCRIPTION
**Pull request by slacalle**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/r-phewas/meta.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ build:
 requirements:
   build:
     - gcc_linux-64
-    - gofrtran_linux-64
+    - gfortran_linux-64
```
Comment (marcelotrevisani) [inline recipes/r-phewas/meta.yaml]: it would be great if you change that to:

```
- {{ compiler('c') }}
- {{ compiler('fortran') }}
```

:)

Comment (slacalle) [reply recipes/r-phewas/meta.yaml]: Hi @marcelotrevisani I tried with jinja functions but did not work. will try again.

File: recipes/r-phewas/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  skip: True    # [win]
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```
Comment (marcelotrevisani) [inline recipes/r-phewas/meta.yaml]: I think we have to change that to:

```
- {{ compiler('r_c') }}
```

Comment (slacalle) [reply recipes/r-phewas/meta.yaml]: Hi @marcelotrevisani ,

From what I gather from https://bioconda.github.io/troubleshooting.html it looks something to do with the build environment and conda-build=3 new features. i will look to ammend the recipe to be compatible with the former version.

Thanks,

**Pull request closed without merging**